tests/redisUnit.js

```diff
@@ -0,0 +1,199 @@
+function quitDB(client) {
+  return new Promise((resolve) => {
+    client.flushdb(() => {
+      // never fails
+      resolve();
+    });
+  })
+  .then(() => new Promise((resolve) => {
+    client.quit(() => {
+      // do not check for errors, just resolve
+      resolve();
+    });
+  }));
+}
```

Comment:
I've got no idea if that's what people wanted when designed the feature, but this one-liner works well :)
`return new Promise((resolve) => client.flushdb(() => client.quit(() => resolve())));`